packages/components/organisms/f-header/CHANGELOG.md

```diff
@@ -4,6 +4,19 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 
+v7.4.0
+------------------------------
+*December 9, 2021*
+
+### Changed
+- Fix some display issues for highlight theme
+  - Make icons visible in mobile nav.
+  - Reduce header height for mobile viewports.
```

Comment:
I think I know why you've adjusted the height (as it looked broken and too big) but I think this is used (maybe on search?) so that an extra input text field is covered inside the header potentially (if you look on SERP live it shows this on mobile)

It may be worth providing a prop that extends the height for this use case, as atm I guess it's just being made the default as the highlight header is only ever used on SERP.

Reply:
Added a new prop to make the header "tall below mid", and added a visual test to cover that.